layouts/partials/header.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
```

Comment:
@AustinLewis1 is all of this content here just so we can change that one icon from the "forking" to the "edit pencil"?

Reply:
Yes it is